plugins/in_netif/in_netif.c

```diff
@@ -94,45 +94,23 @@ static int init_entry_linux(struct flb_in_netif_config *ctx)
 }
 
 static int configure(struct flb_in_netif_config *ctx,
-                     struct flb_input_instance *in,
-                     int *interval_sec,
-                     int *interval_nsec)
+                     struct flb_input_instance *in)
 {
-    const char *pval = NULL;
+    int ret;
     ctx->map_num = 0;
 
-    /* interval settings */
-    pval = flb_input_get_property("interval_sec", in);
-    if (pval != NULL && atoi(pval) >= 0) {
-        *interval_sec = atoi(pval);
-    }
-    else {
-        *interval_sec = DEFAULT_INTERVAL_SEC;
-    }
-
-    pval = flb_input_get_property("interval_nsec", in);
-    if (pval != NULL && atoi(pval) >= 0) {
-        *interval_nsec = atoi(pval);
-    }
-    else {
-        *interval_nsec = DEFAULT_INTERVAL_NSEC;
+    /* Load the config map */
+    ret = flb_input_config_map_set(in, (void *)ctx);
+    if (ret == -1) {
+        return -1;
```

Comment:
Could you output log ?
If `flb_input_config_map_set` failed, fluent-bit will exit without error logs.